test/Microsoft.Dnx.Testing.CommonUtils/Exec.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text;
+
+namespace Microsoft.Dnx.Testing
+{
+    public class ExecResult
+    {
+        public int ExitCode { get; set; }
+        public string StandardOutput { get; set; }
+        public string StandardError { get; set; }
+
+        public ExecResult EnsureSuccess()
+        {
+            if (ExitCode != 0)
+            {
+                throw new InvalidOperationException($"Exit code was {ExitCode}");
+            }
+
+            return this;
+        }
+    }
+
+    public class Exec
+    {
+        public static ExecResult Run(
+            string program,
+            string commandLine,
+            Action<Dictionary<string, string>> envSetup = null,
+            string workingDir = null)
+        {
+            Console.WriteLine($"Running: {program} {commandLine}");
```

Comment:
Which ones are you referring to? The only similar helper I found was in the current CommonTestUtils project but I believe the idea is to replace the current test infrastructure so we can't use that one.